### ISSUE TITLE
Help commands have bad performance

### ISSUE THREAD

**Issue by bergmeister** [Microsoft/vsts-cli#95]
Trying to get help takes about a second, which is a very noticeable lag and given that it should not need to call a REST endpoint for that unlike other commands, I do not expect it to take so long.
In PowerShell (on a 4 core Xeon VM) for example:
````powershell
> Measure-Command { vsts -h }
Days              : 0
Hours             : 0
Minutes           : 0
Seconds           : 0
Milliseconds      : 907
Ticks             : 9077662
TotalDays         : 1.05065532407407E-05
TotalHours        : 0.000252157277777778
TotalMinutes      : 0.0151294366666667
TotalSeconds      : 0.9077662
TotalMilliseconds : 907.7662

````

**Comment by tedchamb** [comment]
We have already made some improvements to speed of help commands in the dev branch, and we will be investigating to see what else we can do.

**Comment by bergmeister** [comment]
Thanks. How often are you planning to release the next previews? It would be great to get the installers from the build or have a URL pointing to a nightly build.

**Comment by tedchamb** [comment]
The current thinking is that we would put a new preview build out each sprint (3 weeks) if there are enough changes to warrant it.  That may change.  We also plan to have a page where you can download nightly dev builds.  I don't have an estimate yet on when that will be ready.

**Comment by tedchamb** [comment]
Version 0.1.1 of the cli is now available, which contains the change in #84.

**Comment by tedchamb** [comment]
Done:
https://github.com/Microsoft/vsts-cli/releases/tag/0.1.1
